tests/acceptance/config/behat.yml

```
@@ -253,6 +253,15 @@ default:
         - WebDavLockingContext:
         - PublicWebDavContext:
 
+    apiShare:
```

Should we specify the suite as ng? so that we wont get confused with the old share tests
```suggestion
    apiSharingNg:
```